Docs/source/install/hpc/cori.rst

```diff
@@ -153,16 +164,19 @@ Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following comman
    cmake -S . -B build -DWarpX_OPENPMD=ON -DWarpX_DIMS=3
    cmake --build build -j 16
 
-The general :ref:`cmake compile-time options and instructions for Python (PICMI) bindings <building-cmake>` apply as usual.
+The general :ref:`cmake compile-time options and instructions for Python (PICMI) bindings <building-cmake>` apply as usual:
 
-.. warning::
+.. code-block:: bash
 
-   Consider that all three Cori partitions are *incompatible*.
+   # PICMI build
+   cd $HOME/src/warpx
 
-   Do not *source* multiple ``...warpx.profile`` files in the same terminal session.
-   Open a new terminal and log into Cori again, if you want to switch the targeted Cori partition.
+   # update Python dependencies
+   python3 -m pip install --user -U pip
+   CFLAGS="$(CC --cray-print-opts=cflags)" LDFLAGS="$(CC --cray-print-opts=libs)" python3 -m pip install -U --user mpi4py
```

Comment:
Nersc documents: https://docs.nersc.gov/development/languages/python/parallel-python/#mpi4py-in-your-custom-conda-environment
```
MPICC="cc -shared" python3 -m pip install -U --user mpi4py
```

which works as well.

We can also add `--no-cache-dir` here in case a user already tried to build `mpi4py` (even implicitly)

Reply:
Fixed :)